RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 -----
 * [*] [internal] Blogging Reminders notifications and scheduling can be turned off for WordPress as part of Jetpack focus behind a feature flag. [#19611]
 * [*] [internal] Weekly Roundup notifications and scheduling can be turned off for WordPress as part of Jetpack focus behind a feature flag. [#19590]
+* [*] [internal] Remote notifications are turned off for WordPress when moving to Jetpack behind a feature flag. [#19616]
```

Comment:
nitpick:

> behind a feature flag

The wording here doesn't flow too well IMO, but it is an internal release note nad other release notes use this format.

We don't have a convention for marking this as being disabled behind a feature flag. Perhaps something like this would be clearer:

```suggestion
* [*] [internal] When a user migrates to the Jetpack app and allows notifications, WordPress app notifications are disabled. This is released disabled and is behind a feature flag. [#19616]
```

Reply:
Yes, I agree that it doesn't flow well grammatically. I just wanted to keep it as compact as possible while also mentioning that it's behind a feature flag. I saw a similar formulation in other parts of the release notes. 

Thank you for the suggestion, lets make it more clear 👍